analyses/cell-type-ewings/template_notebooks/cnv-workflow/03-copykat.Rmd

```diff
@@ -47,8 +47,9 @@ knitr::opts_chunk$set(
 
 
 ```{r base paths}
-# The base path for the OpenScPCA repository, found by its (hidden) .git directory
-repository_base <- rprojroot::find_root(rprojroot::is_git_root)
+# The base path for the OpenScPCA repository, found by LICENSE.md file
+# can't use .git because GHA doesn't find .git
+repository_base <- rprojroot::find_root(rprojroot::has_file("LICENSE.md"))
```

Comment:
This seems fine to me. Something else you could probably use is the `.github` directory, or you could even be kind of cutely recursive and use `rprojroot::find_root(rprojroot::has_dir("analyses/cell-type-ewings")) 

Alternatively, you could find the `module_base` first with `rprojroot::find_root(rprojroot::is_renv_project)` and then define the repository base two levels up from that. I think that is where I would lean toward?

To avoid adding `..` to the paths, you can use `repository_base <- dirname(dirname(module_base))`